.eslintrc.js

```diff
@@ -13,5 +13,6 @@ module.exports = {
   rules: {
     eqeqeq: "error",
     "prefer-const": "error",
+    curly: "error",
```

Comment:
I would vote for ["error", "multi-line", "consistent"]. Some simple short lines look straightforward to me.

Reply:
I tend to lean towards explicit brackets for conditionals, for a couple reasons:

It provides consistency everywhere, where you don't have to _think_ about how to write an if statement (an if statement will _always_ have opening and closing brackets)

It provides a somewhat cleaner git history. For example, if I write code like...
```js
if (someCheck) return value;
```
and you update the code to...
```js
if (someCheck) {
  const calculatedValue = value + 1;
  return calculatedValue;
}
```
then it _appears_ that _you_ were responsible for defining `someCheck` (because you edited that line of code). So if a future dev has a question about `someCheck`, they might be more likely to reach out to you, although I was the one responsible for the original code. It's a super nitpicky edge case, though.

Also, I just find something like this kind of gross (but valid with w/ `["error", "multi-line", "consistent"]`):
```js
let sum = 0;
while (item.next()) sum += item.value;
```

I'd love to get other people's feedback on this one. I could definitely be over thinking it 😁

Reply:
Makes sense. I agree with you now:) Thank you

Reply:
Agreed with @scnwwu. Looks good here